map/lanelet2_extension/lib/visualization.cpp

```diff
@@ -929,6 +932,9 @@ visualization_msgs::msg::MarkerArray visualization::laneletsBoundaryAsMarkerArra
   if (!center_line_strip.points.empty()) {
```

Comment:
[IMO] 
I think replace this center line strip to arrow marker would be smarter, isn't it?

Reply:
@TakaHoribe How about it?

Reply:
My opinion is: the linestring is useful to understand the order of the points, so it would be nice if we can have both (linestring and points). Then, we can keep the implementation as this is.

Reply:
sorry for late replay
arrow marker is also
- useful to understand the order of the points
- position of each center line point.
https://github.com/autowarefoundation/autoware.universe/pull/501

if you want center line fast, I approve this change

Reply:
@taikitanaka3 
![image](https://user-images.githubusercontent.com/9547070/169444989-ca6c58b8-98ba-46a6-9469-5b04156744a5.png)
This way?

Reply:
great!